app/mm-redux/utils/deep_freeze.ts

```diff
@@ -58,7 +58,7 @@ function throwOnImmutableMutation(key: string, value: any) {
     throw Error(
         'You attempted to set the key `' + key + '` with the value `' +
         JSON.stringify(value) + '` on an object that is meant to be immutable ' +
-        'and has been frozen.'
+        'and has been frozen.',
```

Comment:
Could this comma trigger a runtime error?

Reply:
it is enforced by  eslint, so no